website/docs/docs/dbt-cloud/access-control/access-control-overview.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "Access Control"
+id: "access-control-overview"
+---
+
+## Overview
+
+dbt Cloud administrators can use dbt Cloud's permissioning model to restrict the
+permissions of users on their account. This access control comes in two
```

Comment:
```
dbt Cloud administrators can use dbt Cloud's permissioning model to restrict the
permissions of users on their account.
```

How about this instead?
`dbt Cloud administrators can use dbt Cloud's permissioning model to restrict users' access on their account.`